rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -898,6 +898,30 @@ void getActiveFederationCreationBlockHeight_afterRSKIP186AndNoValueInStorage_ret
 
     }
 
+    @Test
+    void setActiveFederationCreationBlockHeight() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP186)).thenReturn(true);
```

Comment:
Same comment as in previous PR regarding activations

Reply:
I accidentally included these changes in the branch it was pointing to (previous pr). Updated.